### ISSUE TITLE
Bug In VM?: SIGSEGV: Illegal storage access. (Attempt to read from nil?)

### ISSUE THREAD

**Issue by loloicci** [loloicci/nimly#24]
Sometimes ``SIGSEGV: Illegal storage access. (Attempt to read from nil?)`` is issued in compile-time.
Maybe it is bug in VM (and not in nimly).
https://dev.azure.com/oxisccl/nimly/_build/results?buildId=29

**Comment by loloicci** [comment]
See also:
https://github.com/loloiccl/nimly/commit/1128e18dbcb268d4f78450fbbc67870c8bff014a
(After this commit, test fails)

**Comment by loloicci** [comment]
maybe sets.incl has some bugs with VM.

nimly: ``3ca678e22da3fc0607be3a4a875f77859e5f8524``
Nim:  ``fabc2a7086bb83afc2abfd6d5841f25c8387609f``

Sometimes following error is issued when ``nimy`` is used.

```stack trace: (most recent call last)
test_nimly.nim(51, 5)
test_nimly.nim(51, 5)    :tmp
../src/nimly/lalr.nim(162, 18) makeTableLALR
../src/nimly/lalr.nim(110, 16) toLALRKernel
../../Nim/lib/pure/collections/sets.nim(143, 24) incl
test_nimly.nim(33, 6) template/generic instantiation of `nimy` from here
../../Nim/lib/pure/collections/sets.nim(143, 24) Error: attempt to access a nil address
```

**Comment by loloicci** [comment]
Maybe fixed in Nim v1.0.0.

**Comment by loloicci** [comment]
see also #30
